backend/controllers/login.controller.js

```diff
@@ -0,0 +1,38 @@
+const { User } = require("../models/user.model");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const verifyUserCredentials = async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+    const user = await User.findOne({ email });
+    if (!user) {
+      return res
+        .status(401)
+        .json({ success: false, errorMessage: "User doesn't exist!" });
+    }
+
+    const isPasswordMatch = await bcrypt.compare(password, user.password);
+    if (isPasswordMatch) {
+      const token = jwt.sign(
+        {
+          userId: user._id,
+          expiry: "24h",
+        },
```

Comment:
Why you created a separate file for login when you already have a user.controller.js file. You put the logic of signup there why not login too?